modules/ossm-about-migrating-to-cluster-wide.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-about-migrating-to-cluster-wide_{context}"]
+= About migrating to a cluster-wide mesh
+
+
+In a cluster-wide mesh, one `ServiceMeshControlPlane` (SMCP) monitors all of the resources for an entire cluster. You can migrate an existing deployment from multitenant mesh to a cluster-wide mesh using {SMProductName} version 2.5 or later. If a deployment must have more than one SMCP, then you cannot migrate the deployment to a cluster-wide mesh.
+
+When you change the SMCP from multitenant to cluster-wide, Istiod watches all namespaces with with the `istio-discovery: enabled` label that reside in the cluster. 
+
+Namespaces with the label applied are accessible through the network, and an application can call a service that resides in one of the available namespaces. Although these namespaces are available to the network, they do not receive sidecar injection. The `ServiceMeshMemberResource` specifies the namespaces that receive sidecar injection.
+
+After migrating to a cluster-wide mesh, you do not have to restart applications if you use the same version of Istio. The application proxies will connect to the new cluster-wide control plane, and work the same way they did for a multitenant deployment. If you migrate to a different version of Istio, then you must restart applications. 
+
+If you only change the mode then you do not have to restart the applications. If you make any other changes, such as migrating to a different Istio version, then you must restart the applications.
```

Comment:
Hi, I think this last paragraph is a little redundant with the previous one. Maybe we can say something like:
"Restart of the applications is only needed when the Istio version changes"

Additionally, we mentioned Istio version changes, but it's not better to refer to maistra version? I do not know if all the user know when we have a bump of version for Istio.

Reply:
@fjglira I've paraphrased this based on your suggestion.

@jewertow when you review this PR, I'd be curious to know your preference for Istio versus Maistra as I think through this.

Reply:
Jacek and I chatted and think using Istio might be better because in the code Istio is usually what is seen: Istio proxy etc.